### ISSUE TITLE
grub update reset setting (Bugzilla Bug 1001)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#1001]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 1001
Date: 2014-11-26 20:30:04 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;
To:   OpenMandriva QA &lt;<bugs@openmandriva.org>&gt;
CC:   @benbullard79, @tapwag

Duplicates:   #1483
See also:     https://issues.openmandriva.org/show_bug.cgi?id=1594
See also:     https://issues.openmandriva.org/show_bug.cgi?id=1483
Last updated: 2016-04-05 16:20:28 +0000

## Comment 7561
Date: 2014-11-26 20:30:04 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

In dual boot system, Windows 7 and OpenMandriva, I set Windows as Default Entry. When I update to grub2-2.00-40-omv2014.0-x86_64 Default Entry is reset to default (OpenMandriva)

Similar to bug #990.

## Comment 7564
Date: 2014-11-27 04:16:41 +0000
From: @benbullard79

Go to Systemsettings>Start up and shut down>Grub2 Bootloader and see if 'The next booted entry will become default' is selected. I believe this is the installers default setting. If so UN-select it. Then select Windows from the drop down list then select apply. You will be asked for administrator password a couple of times. Then from Konsole as root (super user) run:

# grub2-mkconfig -o /boot/grub2/grub.cfg

See if this works. I guess to test you could try to "urpmi --replacepkgs grub2-2.00-40-omv2014.0-x86_64" and "urpmi --replacepkgs distro-kde4-config-OpenMandriva-2014.0-1.11-omv2014.0.noarch" and see if either is changing settings. I'm going to test this myself as I have a notebook dual-booting Win 7, OpenMandriva Lx 2014.1, and openSUSE.

## Comment 7565
Date: 2014-11-28 22:20:01 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

(In reply to comment #1)
> Go to Systemsettings>Start up and shut down>Grub2 Bootloader and see if 'The
> next booted entry will become default' is selected. I believe this is the
> installers default setting. If so UN-select it. 
Yes, this is the default. But I uncheck it when I set bootloader.

>Then select Windows from the
> drop down list then select apply. You will be asked for administrator
> password a couple of times. Then from Konsole as root (super user) run:
> 
> # grub2-mkconfig -o /boot/grub2/grub.cfg
Till here it's the complete bootloader setup. I did it and it works, I check it many times because I boot several times per day.
These setting changes only after updating grub2 package. 

> See if this works. I guess to test you could try to "urpmi --replacepkgs
> grub2-2.00-40-omv2014.0-x86_64" and "urpmi --replacepkgs
> distro-kde4-config-OpenMandriva-2014.0-1.11-omv2014.0.noarch" and see if
> either is changing settings. I'm going to test this myself as I have a
> notebook dual-booting Win 7, OpenMandriva Lx 2014.1, and openSUSE.

# urpmi --replacepkgs grub2-2.00-40-omv2014.0.x86_64
No package named grub2-2.00-40-omv2014.0.x86_64

Note that I changed slightly package name because I make a mistake in description. This name is copied from MCC.

## Comment 7573
Date: 2014-11-30 00:49:29 +0000
From: @benbullard79

I did this:

# rpm -qa grub2
grub2-2.00-40-omv2014.0.x86_64

# urpmi --replacepkgs grub2-2.00-40

    $MIRRORLIST: media/main/updates/grub2-2.00-40-omv2014.0.x86_64.rpm
installing grub2-2.00-40-omv2014.0.x86_64.rpm from /var/cache/urpmi/rpms                                                             
Preparing...                     ###################################################################################################
      1/1: grub

and it does not change settings here. If this issue persists we'll need a developer to get involved. Developers will need for you to do this:

https://forums.openmandriva.org/en/discussion/592/bug-report-tool-for-openmandriva-lx#latest

wish I could be of more help.

## Comment 7585
Date: 2014-11-30 20:31:17 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Retried and this time "replacepkgs" works. 

# urpmi --replacepkgs grub2-2.00-40

    http://abf-downloads.rosalinux.ru/openmandriva2014.0/repository/x86_64/main/updates/grub2-2.00-40-omv2014.0.x86_64.rpm
installazione di grub2-2.00-40-omv2014.0.x86_64.rpm da /var/cache/urpmi/rpms                                                     
In preparazione...               ###############################################################################################
      1/1: grub2                 ###############################################################################################

As for you it don't change settings.

But I can assure that updating change settings, it happened in several (8 or 10) pc with same hardware. 
Not sure but I think I still have one pc not yet updated.

## Comment 7775
Date: 2014-12-23 22:33:21 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

Update to 2.02-1.beta2.4:2014.0 version don't affect grub settings.

## Comment 11978
Date: 2016-02-23 14:46:29 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

It happens again. 
Updating to grub2-2.02-1.beta2.15-omv2014.0.x86_64 Default Entry is reset to default (OpenMandriva).

## Comment 12385
Date: 2016-03-27 20:23:06 +0000
From: Giorgio &lt;<gio_6b@yahoo.it>&gt;

This is not the only package that change boot settings. Closed as this bug is included in bug #1483.

*** This bug has been marked as a duplicate of bug #1483 ***
